newEvaluation.css

```diff
@@ -13,5 +13,91 @@ li
 
 fieldset
 {
-	max-width: 850px;
+	/*max-width: 850px;*/
+	width: 90%;
+}
+
+section
+{
+	width: 90%;
+}
+
+form
+{
+	flex-direction: column-reverse;
```

Comment:
I was thinking that it will align vertically the sections inside the form. I also tried on a div container but apparently i'm not sure yet how to use it.